scss/bootstrap/components/_custom-select.scss

```diff
@@ -11,8 +11,9 @@ $crm-customselect-feedback-padding-right: $crm-customselect-padding-right + 10;
 
   > select {
     background: transparent;
-    border: 1px solid #949caa;
-    padding: 6px #{$crm-customselect-padding-right}px 6px 12px;
+    border: 1px solid $input-border;
+    height: $input-height-base;
```

Comment:
This is done to keep a consistent look across the theme.
`$input-height-base` variable is present for this purpose only and used by `.form-control` class which works with `input`, as this is a `select` adding the height does the same thing.
Previously the height of the select was calculated based on padding, but specifying the height is more reliable.